README.md

```diff
@@ -16,16 +16,15 @@ Our supported platform are:
 - Kubernetes: 1.32, 1.31, 1.30
 - Node OS: Linux with LVM2 (\*1), specifically tested on Ubuntu
 - CPU Architecture: x86 (\*2), arm64 (\*3), ppc64le (\*3), s390x (\*4)
-- Filesystems: ext4, xfs, btrfs\[beta(\*5)\]
+- Filesystems: ext4, xfs, btrfs
 - lvm version 2.02.163 or later (adds JSON output support)
 
-\*1 The host's Linux Kernel must be v4.9 or later which supports `rmapbt` and `reflink`, if you use xfs filesystem with an official docker image.  
-\*2 Tier1 support. The official docker images are provided and all functionalities are tested by CI.  
-\*3 Tier2 support. The official docker images are provided, but no tests run by CI.  
-\*4 Tier3 support. No docker images are provided, and no tests run by CI, but binaries are built.  
-\*5 btrfs is in beta as of v0.31.0. It can be used in production, but has not seen extensive use by our users yet.
+\*1 The host's Linux Kernel must be v4.9 or later which supports `rmapbt` and `reflink`, if you use xfs filesystem with an official docker image.
+\*2 Tier1 support. The official docker images are provided and all functionalities are tested by CI.
+\*3 Tier2 support. The official docker images are provided, but no tests run by CI.
+\*4 Tier3 support. No docker images are provided, and no tests run by CI, but binaries are built.
```

Comment:
The format is broken because you deleted the trailing two white spaces. Please give it back as it was.
```suggestion
\*1 The host's Linux Kernel must be v4.9 or later which supports `rmapbt` and `reflink`, if you use xfs filesystem with an official docker image.  
\*2 Tier1 support. The official docker images are provided and all functionalities are tested by CI.  
\*3 Tier2 support. The official docker images are provided, but no tests run by CI.  
\*4 Tier3 support. No docker images are provided, and no tests run by CI, but binaries are built.  
```